tests/test_pytest/test_plugin_schema/bad/feature_annotators/cartopy.yaml

```diff
@@ -0,0 +1,155 @@
+---
+error: ValidationError
+error_pattern: ".*'spec' is a required property.*"
```

Comment:
@jsolbrig Does the ".*" do something in these error patterns?  Oh, is it actually grepping the error message?

Reply:
This is actually doing a regex.